### PR TITLE
[SRDF]  Fix parsing of reference config + Add ability to read a SRDF stream. 

### PR DESCRIPTION
**Pull request by jmirabel**

### CODE REVIEW

File: unittest/srdf.cpp [hunk 1]
```diff
@@ -60,6 +60,38 @@ BOOST_AUTO_TEST_CASE(readReferenceConfig)
...
+    "<group_state name='reference' group='all'>"
+    "     <joint name='joint'  value='0.0' />"
+    "</group_state>"
+    "</robot>";
+  
+  Model model;
+  pinocchio::urdf::buildModelFromXML(urdf, model);
+
+  std::istringstream iss (srdf);
+  pinocchio::srdf::loadReferenceConfigurations(model,iss,false);
```
Comment (jmirabel) [inline unittest/srdf.cpp]: This was failing before because unbounded revolute joints where not handled properly.

**Comment by jmirabel** [comment]
> A simple thing you can add is the Python binding of this new signature

Can you easily convert a Python object into a std::istream object ? If not, then it is not possible without having a different name.

**Comment by jcarpent** [comment]
See https://github.com/stack-of-tasks/pinocchio/blob/db8fe0add7d10ea31ff13aa677e30fcc3b569e42/bindings/python/parsers/parsers.hpp#L75 for further details.

**Comment by jmirabel** [comment]
The functions have different names in C++. I wanted to avoid that.

**Pull request merged**